Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
+        "description": "Your first steps to set up powerful C++ tools in a lightweight, performant editor!",
+        "steps": [
+          {
+            "id": "verify.compiler.mac",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. [Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isMac",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing Clang on macOS",
+              "path": "walkthrough/install-clang-macos.md"
+            }
+          },
+          {
+            "id": "verify.compiler.linux",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isLinux",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing GCC on Linux",
+              "path": "walkthrough/install-gcc-linux.md"
+            }
+          },
+          {
+            "id": "verify.compiler.windows",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isWindows",
+            "doneOn": {
```

Comment:
@Colengms I'm getting squiggles: "doneOn is deprecated. By default steps will be checked off when their buttons are clicked, to configure further use completionEvents""

Reply:
From Jackson (software developer for VS Code): The new VS Code Insiders "allows checking off based on an extension being installed, link clicked, command run, or the step simply being selected. _The existing doneOn syntax will still work, but it's deprecated in favor of new "completionEvents"._ Also, by default, if neither of those are defined simply pressing any of the buttons on a step will check the step off (this seems to be the common case). (Exception: if the step has no buttons at all, selecting it will check the item off)."

More info and example code here: https://github.com/microsoft/vscode/issues/122570#issuecomment-840209548

Reply:
@Colengms Can we switch to completionEvents?